watertap/unit_models/coag_floc_model.py

```diff
@@ -446,7 +446,7 @@ def fix_tss_turbidity_relation_defaults(self):
         self.intercept.fix()
 
     # initialize method
-    def initialize(
+    def initialize_build(
```

Comment:
Why are we changing to initialize_build for unit models?

Reply:
This is the new way to do things in IDAES - it is needed to handle initialization of costing blocks (and potentially other model add-ons).